js/script.js

```diff
@@ -1,30 +1,36 @@
-// Check if book collection is already stored in local storage
-let bookCollection = JSON.parse(localStorage.getItem('bookCollection')) || [];
+import Book from './Book.js';
+import BookCollection from './BookCollection.js';
+
+const bookCollection = new BookCollection();
 
 // Add book to collection
 const addBook = () => {
   const title = document.getElementById('title-input').value;
   const author = document.getElementById('author-input').value;
-  bookCollection.push({ title, author });
-  localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+
+  bookCollection.addBook(new Book(title, author));
 
   displayBooks(); // eslint-disable-line
 };
 
 // Remove book from collection // Remove duplicates
-const removeBook = (title) => { // eslint-disable-line
-  bookCollection = bookCollection.filter((book) => book.title !== title);
-  localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+
+const removeBook = (title) => {
+  bookCollection.removeBook(title);
   displayBooks(); //eslint-disable-line
```

Comment:
- [ ] Kindly remove disable linters as linters help us to write clean industry acceptable code